changelog/12253.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:improvement
+core: Update github.com/ulikunitz/xz
```

Comment:
would it make sense to link to the security issue here, stating the reason for the upgrade ?

Reply:
Done!